recipes/ecctl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ecctl" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/elastic/ecctl/archive/v{{ version }}.tar.gz
+  sha256: 31c1dfbfd67a037f5895bd65b233ef89e976d7ce0f6d544a12d66e65293d11c6
+
+# Can we get windows to work?
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+    - go-licenses
+    - make
+
+test:
+  requires:
+    - m2-bash       # [win]
+    - m2-coreutils  # [win]
```

Comment:
Why do you need this?

Reply:
It looks like those requirements weren't needed since I'm skipping the windows build anyways, I removed them.